### PR TITLE
Adding optional missed construction parameters to @types/lambda-log v3

### PR DESCRIPTION
**Pull request by SCdF**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://lambdalog.dev/docs/v3/api#logoptions
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by armonge — APPROVED** [review]

**Comment by danielsitnik** [comment]
Please @armonge and @SCdF, do you know what else is need for this PR to be merged?

I'm using LambdaLog 3.1.0 right now and to be able to set some options like `tagsKey` I have to "silence" TS and ESLint like this:
```ts
const logger = new LambdaLog({
  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
  // @ts-ignore
  tagsKey: null,
});
```

I'm not familiar with the with the PR/merge process of thsi repo, appreciate if you can look into this.
Thanks!

File: types/lambda-log/index.d.ts [hunk 1]
```diff
@@ -55,10 +55,14 @@ export interface LambdaLogOptions {
...
-    stdoutStream?: WriteStream | undefined;
-    // Optional stream to write stderr messages to
-    stderrStream?: WriteStream | undefined;
+    // Console-like object containing all standard console functions. Allows logs to be written to any custom location. Default is console.
+    logHandler?: Console;
+    // Override the key in the JSON log message for the message. Default is "msg".
+    levelKey?: string | null;
+    // Override the key in the JSON log message for the message. Default is "msg".
+    messageKey?: string;
+    // Override the key in the JSON log message for the tags. Set to null to remove this key from the JSON output. Default is "_tags".
```
Comment (SCdF) [inline types/lambda-log/index.d.ts:64]:
```suggestion
    // Override the key in the JSON log message for the tags. Set to null to remove this key from the JSON output. Default is '_tags'.
```

**Comment by SCdF** [comment]
@danielsitnik thanks for reminding me this exists. I have no idea why it hasn't also merged back in October. The "Where is GH Actions?" tag seems to be the problem. I tried making a tiny update, maybe that will help?

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## lambda-log ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lambda-log@latest/))
was missing the following properties:
1. console
1. addLevel
1. result
1. addListener
1. on

<details>
<summary>as well as these 13 other properties...</summary>
<p>once, removeListener, off, removeAllListeners, setMaxListeners, getMaxListeners, listeners, rawListeners, emit, listenerCount, prependListener, prependOnceListener, eventNames</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e84d2f270171a15c8c69db890b4b59531d9a533d
</p>

**Comment by SCdF** [comment]
@armonge no actual changes, I just slightly changed a comment to refire GitHub actions to hopefully trigger the auto merge

**Comment by danielsitnik** [comment]
@armonge can you please review? thanks

**Review by armonge — APPROVED** [review]

**Comment by SCdF** [comment]
Ready to merge

**Pull request merged**